config/initializers/active_admin.rb

```diff
@@ -1,6 +1,9 @@
 Dir[Rails.root.join('lib/extensions/active_admin/*.rb')].each { |f| require f }
 
 ActiveAdmin.setup do |config|
+  # Custom Load paths
+  config.load_paths += [File.join(Rails.root, 'app', 'commands')]
```

Comment:
Hmm, and other not standard paths like `services` should not be added here as well? Is that needed?

Reply:
you're right! I just had to move the file to `command` directory